packages/data-stores/src/wpcom-plans-ui/actions.ts

```diff
@@ -9,4 +9,19 @@ export const resetStore = () =>
 		type: 'WPCOM_PLANS_UI_RESET_STORE',
 	} as const );
 
-export type WpcomPlansUIAction = ReturnType< typeof setShowDomainUpsellDialog | typeof resetStore >;
+export const setStorageAddOnForPlan = ( {
+	addOnSlug,
+	plan,
+}: {
+	addOnSlug: string;
+	plan: string;
+} ) =>
+	( {
+		type: 'WPCOM_PLANS_UI_SET_STORAGE_ADD_ON_FOR_PLAN',
```

Comment:
I initially went with the term add on where possible because it matches the vocabulary we're using for the UX, product, and business side of things, but since this is the data store for plans UI, I think you're right.

Mulling this over, but will probably change the name 🤔

Reply:
https://github.com/Automattic/wp-calypso/pull/80158/commits/d95e8c08f6d2b0989be7ec4333c7dfded49ede93